docs-md/config/modules/http_hmac_cookie.md

```diff
@@ -0,0 +1,57 @@
+
+
+# http_hmac_cookie
+
+The http_hmac_cookie provides a safe way to persent authentication
+information in http sessions.
+
+
+  * **loader**: C
+  * **image**: http_hmac_cookie.so
+
+### Module Configuration
+
+    
+ * **`key`** (optional) 
+
+   allowed: `/^.+$/`
+
+   A hex encoded key, if none is specified a random key will be
+   generated.
+
+ * **`max_age`** (optional)  [default: `86400`]
+
+   allowed: `/^\d+$/`
+
+   The number of seconds the cookie will be valid for (default 1
```

Comment:
Is 0 valid here?

Reply:
"valid" sure, it means auth won't work.

Reply:
0 in cookie life usually means "session" no?  Why wouldn't it work?

Reply:
Quick googling.  0 should be delete, -1 is session. ok